packages/terra-doc-template/examples/test-examples/DefaultDocTemplate.example.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import DocTemplate from '../../src/DocTemplate';
+import readme from './TestReadme.md';
+// eslint-disable-next-line import/no-duplicates
```

Comment:
That's strange. I don't see duplicate transformations. Is it because TestComponentExample is uploaded twice?